### PR TITLE
[skin.info.service] 1.1.0

### PR DESCRIPTION
**Pull request by mikesilvo164**
### Description
Update for Skin Info Service removing python 2 backward compatibility, some urllib fixes, and added an auto close osd option.
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.skin.info.service/Utils.py [hunk 1]
```diff
@@ -0,0 +1,286 @@
+import xbmc
+import xbmcaddon
+import xbmcgui
+import os
+import sys
+import json as simplejson
```
Comment (enen92) [inline script.skin.info.service/Utils.py:6]: can you remove the alias to simplejson and simply replace everything with `json`?

File: script.skin.info.service/addon.xml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	</requires>
+	<extension point="xbmc.python.library"          library="default.py" />
+	<extension point="xbmc.python.script"           library="default.py">
+		<provides>executable</provides>
+	</extension>
+	<extension point="xbmc.addon.metadata">
+		<summary lang="en_GB">Script used by skins to show extended media info.</summary>
+		<description lang="en_GB">This script fetches extended information for several media types (artists, albums, movie sets etc.). Needs skin support.</description>
+		<platform>all</platform>
+		<license>GNU GENERAL PUBLIC LICENSE Version 2, June 1991</license>
```
Comment (enen92) [inline script.skin.info.service/addon.xml:14]: spdx identifier

File: script.skin.info.service/default.py [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+                        self.setMovieDetailsforCategory()
+            elif xbmc.getCondVisibility("Container.Content(years) | Container.Content(genres)"):
+                self.selecteditem = xbmc.getInfoLabel("ListItem.Label")
+                if (self.selecteditem != self.previousitem):
+                    clear_properties()
+                    self.previousitem = self.selecteditem
+                    if (self.selecteditem != "") and xbmc.getCondVisibility("!ListItem.IsParentFolder"):
+                        self.setMusicDetailsforCategory()
+            elif xbmc.getCondVisibility('Window.IsActive(screensaver)'):
+                xbmc.sleep(1000)
```
Comment (enen92) [inline script.skin.info.service/default.py:83]: maybe use xbmc.Monitor().waitForAbort(1) ?

**Comment by mikesilvo164** [comment]
All requested changes implemented 😃

**Pull request merged**